src/Tribe/Customizer.php

```diff
@@ -426,29 +425,28 @@ public function inline_style() {
 			return false;
 		}
 
-		// add customizer styles inline with either main stylesheet is enqueued or widgets
-		if ( wp_style_is( 'tribe-events-calendar-style' ) ) {
-
-			wp_add_inline_style( 'tribe-events-calendar-style', wp_strip_all_tags( $this->parse_css_template( $css_template ) ) );
-			$this->inline_style = true;
-
-			return;
-		}
-
-		if ( wp_style_is( 'tribe-events-calendar-pro-style' ) ) {
+		$sheets = [];
 
-			wp_add_inline_style( 'tribe-events-calendar-pro-style', wp_strip_all_tags( $this->parse_css_template( $css_template ) ) );
-			$this->inline_style = true;
+		/**
+		 * Allow plugins to add themselves to this list
+		 *
+		 * @param array $sheets An array of sheets to search for.
+		 * @param string $css_template String containing the inline css to add.
```

Comment:
```suggestion
		 * Allow plugins to add themselves to this list.
		 * 
		 * @since TBD	 
		 *
		 * @param array $sheets An array of sheets to search for.
		 * @param string $css_template String containing the inline css to add.
```